docs/what-we-do/metaos.md

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
This line needs to be removed to make the `title` in the Frontmatter render as it should.